app/scripts/views/sign_up.js

```diff
@@ -257,6 +258,15 @@ function (Cocktail, _, p, BaseView, FormView, Template, AuthErrors, mailcheck,
       this.navigate('cannot_create_account');
     },
 
+    _isSyncChooser: function () {
+      var chooseSyncCapability = this.broker.getCapabilityByName('choose_what_to_sync');
+      // if this is sync and broker is capable of using choose what to sync on the web v1
+      if (this.relier.isSync() && chooseSyncCapability && chooseSyncCapability.version === 'web_v1') {
```

Comment:
How do you envision new capabilities being added, or how could we change what we already have to use capabilities? Any ideas for new capabilities, or transitioning functionality we already have to a capability?